provisioning/device/samples/ProvisioningDeviceClientTpm/readme.md

```diff
@@ -0,0 +1,18 @@
+# Provisioning Device Client Sample - TPM Attestation
+
+## Overview
+
+This is a quick tutorial with the steps to register a device in the Microsoft Azure IoT Hub Device Provisioning Service using the Trusted Platform Module (TPM) attestation.
+
+## How to run the sample
+
+Ensure that all prerequisite steps presented in [samples](..\) have been performed.
+To run the sample, in command prompt enter:
```

Comment:
Just to clarify, this is a normal cmd command prompt right? Not a developer command prompt for VS 2017?

Reply:
I wanted to avoid that: on Linux there is no dev command prompt...
The `dotnet` tool must be available.